tests/dsl/test_stencil.py

```diff
@@ -112,3 +120,46 @@ def test_domain_size_comparison(
         stencil._validate_quantity_sizes(quantity)
 
     assert warning_mock.call_count == call_count
+
+
+def two_dim_temporaries_stencil(q_out: FloatField) -> None:
+    with computation(FORWARD), interval(0, 1):
+        tmp_2d_fij: FloatFieldIJ = 1.0
+        tmp_2d_fij32: FloatFieldIJ32 = 2.0
+        tmp_3d_fij64: FloatFieldIJ64 = 3.0
+        tmp_3d_iij: IntFieldIJ = 4
+        tmp_3d_iij32: IntFieldIJ32 = 5
+        tmp_3d_iij64: IntFieldIJ64 = 6
+
+    with computation(PARALLEL), interval(...):
+        q_out = (
+            tmp_2d_fij
+            + tmp_2d_fij32
+            + tmp_3d_fij64
+            + tmp_3d_iij
+            + tmp_3d_iij32
+            + tmp_3d_iij64
+        )
+
+
+@pytest.mark.parametrize(
+    "extent,dimensions,domain,call_count",
+    [
+        ((2, 2, 5), ["x", "y", "z"], (2, 2, 5), 0),
+    ],
+)
+def test_stencil_2D_temporaries(
```

Comment:
can we just not parametrize if we don't need to parametrize? not only because `call_count` is unused ... 

```suggestion
def test_stencil_2D_temporaries(
```